portfolio/pom.xml

```diff
@@ -23,6 +23,18 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
```

Comment:
nit - i think the [latest version is 4.13](https://search.maven.org/artifact/junit/junit)